content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -219,9 +220,29 @@ Run:
 kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
 ```
 
-### Pipeline samples that require "user-gcp-sa" secret
+### Securing the cluster with fine-grained GCP permission control
 
-If sample pipeline requires a "user-gcp-sa" secret, you could create one by:
+#### Workload Identity
+
+> Workload identity is the recommended way for your GKE applications to consume services provided by Google APIs. You accomplish this by configuring a Kubernetes service account to act as a Google service account. Any Pods running as the Kubernetes service account then use the Google service account to authenticate to cloud services.
+
+Referenced from [workload identity documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity). Please read this doc for detailed introduction.
+
+To bind workload identities for kubernetes service accounts (KSAs) provided by pipelines standalone deployment, we provide some helper scripts to set up workload identity bindings for pipelines workloads.
```

Comment:
s/kubernetes/Kubernetes

Here and a couple of times below.

Reply:
Thanks! Fixed